lightning/src/ln/invoice_utils.rs

```diff
@@ -465,20 +464,20 @@ where
 
 				if new_now_public || new_channel_preferable {
 					log_trace!(logger,
-						"Preferring counterparty {} channel {} (SCID {:?}, {} msats) over {} (SCID {:?}, {} msats) for invoice route hints",
+						"Preferring counterparty {} channel (SCID {:?}, {} msats) over {} (SCID {:?}, {} msats) for invoice route hints",
 						log_pubkey!(channel.counterparty.node_id),
-						&channel.channel_id, channel.short_channel_id,
+						 channel.short_channel_id,
 						channel.inbound_capacity_msat,
 						&entry.get().channel_id, entry.get().short_channel_id,
 						current_max_capacity);
 					entry.insert(channel);
 				} else {
 					log_trace!(logger,
-						"Preferring counterparty {} channel {} (SCID {:?}, {} msats) over {} (SCID {:?}, {} msats) for invoice route hints",
```

Comment:
The word "over" got deleted, may need to tweak the log slightly to keep the original meaning as well

Reply:
Reverted these two log lines. As they are making a comparison, it may be a bit weird to rely on the structured field only.